### DISCUSSION TITLE
Parse title into actual title and have the extras as tags

### DISCUSSION THREAD

**Discussion by ChronoStriker1** [Difegue/LANraragi#577] [Q&A] [answered]
Since generally uploads to e-hentai and by extension chaika have standard title name formats, would it be possible to have lanraragi display just the parsed title name?  And at that point, if the full title has things like the convention it was sold at and other optional things (like [Digital], [Scanlation team], etc are converted to tags? Those tags would probably need manual tweaking but a list of known ones or a way of maintaining a list of known ones so you only need to edit those once to be recognized.

**Comment by Difegue** [comment, marked as answer]
This already exists through the [RegexParse](https://github.com/Difegue/LANraragi/blob/dev/lib/LANraragi/Plugin/Metadata/RegexParse.pm) plugin, so enabling it in Plugin Configuration will do what you ask for all incoming files.  

This used to be enabled by default, but since it can fuck up metadata for files that don't come with the standard format it's optional for now.  

Hopefully all those options can be better exposed once I add a proper OOTB screen for first installs (#297)
